### DISCUSSION TITLE
Snapcraft store seems stuck on version 6.7.1

### DISCUSSION THREAD

**Discussion by BioSs54** [ferdium/ferdium-app#1800] [Q&A] [answered]
Hi,

I'm currently using Ubuntu 22.04.4 LTS and I rely on Snap to download and update Ferdium. The problem is that the Snapcraft store seems stuck on version 6.7.1, whereas the latest version is 6.7.4.

Am I the only one experiencing this issue, or are others affected as well?

**Comment by Alphrag** [comment, marked as answer]
Hi, sorry we didn't notice that since 6.7.2 the stable versions have also been published on the `latest/edge` channel instead of the `latest/stable` one. This came from the fact that the variables we are passing to the publishing step are not used correctly anymore, so it is either coming from a change of the snapcraft cli, or from the dependency that calls it automatically in our build process (sitting three layers down from us through `electron-builder`, which is why we didn't pick up that change).

We'll try to sort that out quickly, and move the actual stable versions back into their appropriate channel.

**Comment by Alphrag** [comment]
The version on the stable channel of the snap store is now `6.7.5` and we'll try to make sure it updates properly from now on 🙂
